app/renderer/js/pages/preference/general-section.ts

```diff
@@ -130,11 +138,20 @@ export default class GeneralSection extends BaseSection {
 				<div class="title">${t.__('Reset Application Data')}</div>
                 <div class="settings-card">
 					<div class="setting-row" id="resetdata-option">
-						<div class="setting-description">${t.__('This will delete all application data including all added accounts and preferences')}
+						<div class="setting-description">${t.__('This will reset all application settings to default preserving added organisations and accounts')}
```

Comment:
These strings aren't great; we use American English and the "This will" is weird.  How about:

"Reset all desktop application settings to defaults, preserving added organizations and accounts"

Reply:
Sure thing. Please have a look :)